src/cartridges/int_adyen_SFRA/cartridge/adyen/utils/adyenHelper.js

```diff
@@ -98,25 +98,12 @@ let adyenHelperObj = {
         : new Money(0, product.getPriceModel().getPrice().getCurrencyCode());
       shippingCost = shippingCost.add(productShippingCost);
     })
-    shippingCost = shippingAddress ? shippingCost.addRate(adyenHelperObj.getShippingTaxRate(shippingMethod, shippingAddress)) : shippingCost;
```

Comment:
Can we remove also the const { shippingAddress } in line 89 as it's not utilized anymore?